change/@ni-nimble-angular-f4b9329f-5210-4124-91ae-a2d8840553b7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Looks like [your current PR build](https://github.com/ni/nimble/actions/runs/4633386800/jobs/8198556183?pr=1167) failed with an intermittent test unrelated to your PR. Can you make sure we have a bug tracking it? There are a couple recent ones filed that you can use as a template.

Reply:
Filed #1172.